### PR TITLE
ignorefirstrecord fix

### PR DESCRIPTION
**Pull request by ronmonetaMicro**
#### Pull Request Description

_[Add a description of your pull request here]_

---

#### Future Release Comment
_[Add description of your change, to include in the next release]_
_[Delete any or all irrelevant sections, e.g. if your change does not warrant a release comment at all]_

**Breaking Changes:**
- None

**Features:**
- None

**Fixes:**
- None

### CODE REVIEW

File: azure-kusto-ingest/azure/kusto/ingest/ingestion_properties.py [hunk 1]
```diff
@@ -211,6 +212,7 @@ def __init__(
         self.ingest_by_tags = ingest_by_tags
         self.drop_by_tags = drop_by_tags
         self.flush_immediately = flush_immediately
+        self.ignore_first_record = ignore_first_record
```
Comment (yogilad) [inline azure-kusto-ingest/azure/kusto/ingest/ingestion_properties.py:215]: The name of the ingestion properties json rep for this value is 'ignoreFirstRecord'
What makes this be written correctly when sending the request.

Can we add an E2E test to make sure it works as expected?
there are already E2Es ingesting CSVs so it should be simple

File: quick_start/utils.py [hunk 2]
```diff
@@ -207,9 +218,10 @@ def ingest_from_file(
             :param table_name: Table name
             :param file_path: File path
             :param data_format: Given data format
+            :param ignore_first_record: Flag noting whether to ignore the first record in the table
```
Comment (yogilad) [inline quick_start/utils.py:221]: It should be made clear this is relevant for tabular textual formats (CSV and the likes).
You can also point to the below documentation
https://docs.microsoft.com/en-us/azure/data-explorer/ingestion-properties

File: azure-kusto-ingest/azure/kusto/ingest/ingestion_properties.py (hunk 1 above)
Comment (ronmonetaMicro) [reply azure-kusto-ingest/azure/kusto/ingest/ingestion_properties.py:215]: Not sure i understood correctly - this attribute will receive data from outside, not directly from the JSON - so the naming here doesnt really matter (as long as the JSON is being read correctly somewhere else). did you mean something different?

Comment (AsafMah) [reply azure-kusto-ingest/azure/kusto/ingest/ingestion_properties.py:215]: Hey yochai, we are using the library inflection to convert the case from camelcase to snakecase - https://github.com/Azure/azure-kusto-python/blob/ronmoneta/ignorefirestrecord/quick_start/sample_app.py#L39

We don't need e2e tests here I think, or if we did - this change doesn't warrants them.

I'm resolving

**Review by AsafMah — APPROVED** [review]

**Pull request merged**